### ISSUE TITLE
Joomla 5 compatibility error

### ISSUE THREAD

**Issue by mapexer** [gantry/gantry5#3270]
When testing out websites on Joomla5 - Gantry errors out when the backwards compatibility plugin is disabled, with the message: _Class "JFactory" not found_

**Comment by N8Solutions** [comment]
@mapexer Yes, that is because Gantry 5 is not yet fully compatible with Joomla 5 so you need to leave the backwards compatibility plugin enabled for now.

**Comment by kun18911** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.
